packages/react-i18n/src/babel-plugin/index.ts

```diff
@@ -115,6 +117,12 @@ export default function injectWithI18nArguments({
           ).name;
           const {filename} = this.file.opts;
 
+          if (typeof filename !== 'string') {
```

Comment:
According to `BabelFile` a filename may be undefined/null - such as when you get babel to parse content from stdin.  Here we add a typeguard to ensure that the filename is a string.